cibyl/sources/elasticsearch/client.py

```diff
@@ -0,0 +1,53 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import logging
+
+from elasticsearch import Elasticsearch
+
+from cibyl.exceptions.elasticsearch import ElasticSearchError
+
+LOG = logging.getLogger(__name__)
+
+
+class ElasticSearchClient:
+
+    def __init__(self: object,
+                 host: str = "http://localhost",
+                 port: int = 9200) -> None:
+        self.address = f"{host}:{port}"
+
+    def connect(self: object) -> Elasticsearch:
+        """This is used for connecting to the ElasticSearch instance.
+
+        :param host: Elasticsearch host including the scheme
```

Comment:
The documentation of the host and port parameters should be moved to the init

Reply:
You're right. I also changed the raises to the personalized exception `ElasticSearchError`. Thx.